tests/routers/test_notifiers_security.py

```
@@ -0,0 +1,152 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.pool import StaticPool
+
+from app.database import Base, get_db
+from app.main import app
+from app.models import User
+from app.security import hash_password
+from app.config import settings
+
+# Set required environment variables for testing
+os.environ["SECRET_KEY"] = "test-secret-key"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)